apps/docs/src/app/core/component-docs/shellbar/examples/shellbar-collapsible-example.component.html

```diff
@@ -39,6 +39,63 @@
                 [maxHeight]="'250px'"
             >
             </fd-combobox>
+
+            <fd-user-actions-menu [user]="user">
+                <fd-user-actions-menu-item
+                    [title]="'Looonnngg title Looonnngg title Looonnngg title Looonnngg title Looonnngg title Looonnngg title Looonnngg title'"
```

Comment:
Nit: Maybe it will be better to use something more readable? Default `Lorem ipsum...` :)?